Leanplum-SDK/Classes/LPMessageTemplates.m

```diff
@@ -94,31 +84,29 @@ - (id)init
 // Please give us suggestions for other types of messages!
 - (void)defineActions
 {
-
-    [[[LPAlertMessageTemplate alloc] init] defineActionWithContexts:_contexts];
-    [[[LPConfirmMessageTemplate alloc] init] defineActionWithContexts:_contexts];
-    [[[LPInterstitialMessageTemplate alloc] init] defineActionWithContexts:_contexts];
-    [[[LPWebInterstitialMessageTemplate alloc] init] defineActionWithContexts:_contexts];
-    [[[LPOpenUrlMessageTemplate alloc] init] defineActionWithContexts:_contexts];
-    [[[LPPushAsktoAskMessageTemplate alloc] init] defineActionWithContexts:_contexts];
-    [[[LPRegisterForPushMessageTemplate alloc] init] defineActionWithContexts:_contexts];
-    [[[LPCenterPopupMessageTemplate alloc] init] defineActionWithContexts:_contexts];
-    [[[LPHtmlMessageTemplate alloc] init] defineActionWithContexts:_contexts];
-    [[[LPAppRatingMessageTemplate alloc] init] defineActionWithContexts:_contexts];
-    [[[LPIconChangeMessageTemplate alloc] init] defineActionWithContexts:_contexts];
+    [LPCenterPopupMessageTemplate defineAction];
```

Comment:
I moved away from the pattern of static previously and do not think we should be going back to that

Reply:
This just defining of actions, i purposely left it like we had before to better align previous state. The templates are just defining an action that is going to be executed and show appropriate UI. There is no need to have non static methods here as its just adding an action with callback to internal state which will be executed when its triggered.